### DISCUSSION TITLE
[Bug]: cpf only field return CPF + CNPJ validation message error

### DISCUSSION THREAD

**Discussion by IgordeOliveira** [leandrocfe/filament-ptbr-form-fields#57] [Q&A]
### What happened?

quando uso somente cpf ta retornando mensagem de cpf + cnpj
![image](https://github.com/leandrocfe/filament-ptbr-form-fields/assets/19624377/2b1c6c43-e301-4f67-ac07-8a3d27b50ab4)

### How to reproduce the bug

                                Document::make('cpf')
                                    ->label("CPF")
                                    ->validationAttribute('CPF')
                                    ->cpf()
                                    ->unique(ignorable: fn ($record) => $record)
                                    ->required(),

### Package Version

3.0

### PHP Version

8.1

### Laravel Version

10

**Comment by reneperezdev** [comment]
Aqui estou com o mesmo problema, quando uso somente CPF, está retornando mensagem de cpf + cnpj.
